ui/src/app/pages/profile/data-use-agreement.spec.tsx

```diff
@@ -18,6 +18,25 @@ describe('DataUseAgreement', () => {
     expect(wrapper).toBeTruthy();
   });
 
+  it('should not allow DataUseAgreement without identical initials', () => {
+    const wrapper = component();
+    expect(wrapper.find('[data-test-id="submit-dua-button"]').prop('disabled')).toBeTruthy();
+
+    // fill required fields
+    wrapper.find('[data-test-id="dua-name-input"]')
+      .simulate('change', {target: {value: 'Fake Name'}});
+    // add initials to just one initials input field.
+    wrapper.find('[data-test-id="dua-initials-input"]').first().simulate('change', {target: {value: 'XX'}});
```

Comment:
If i understand this correctly this is testing that the submit button should be disabled if only one initials is added? If yes this feels like a different test case. For test case 'without identical initials' populating each input within forEach is apt